packages/actor-rdf-resolve-hypermedia-qpf/lib/ActorRdfResolveHypermediaQpf.ts

```diff
@@ -14,6 +14,7 @@ implements IActorRdfResolveHypermediaQpfArgs {
   public readonly predicateUri: string;
   public readonly objectUri: string;
   public readonly graphUri?: string;
+  private searchForm: ISearchForm;
```

Comment:
Actor should (almost) never be stateful. Maintaining a state like this can cause weird problems when test/run are run out-of-order.
Instead, you should re-determine the searchForm in both test and run.